03-Mains/Heaving_Water_mass_transformation_decomposition.ipynb

```diff
@@ -0,0 +1,2108 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def var_domain(var):
</pre><p>A more descriptive function name would be helpful. Perhaps <code>subset_by_time_and_space</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>